README.md

```diff
@@ -164,65 +280,73 @@ services:
 
-  # define environment variables here (do not define secrets)
-  variables:
-    FOO: BAR
-    PROJECT_NAME: ${PRE_DEFINED_ENV_VAR}-plus-some
+Example:
+```
+# .servicer/.env.yaml
+AWS_DEFAULT_REGION: us-east-2
+BRANCH: env-test
+BUILD_NUMBER: '0001'
+DATABASE_NAME: my_db
+DATABASE_USERNAME: postgres
+DATABASE_PASSWORD: xxxxxxxx
+GCLOUD_KEY_FILE_PATH: /Users/me/my_gcloud_credentials.json
+GCLOUD_COMPUTE_REGION: us-central1
+GCLOUD_COMPUTE_ZONE: us-central1-f
+PROJECT_NAME: my-project
+SERVICE_ENVIRONMENT: test
 ```
 
-### Git Integration ###
-Servicer provides a few handy Git integrations to facilitate your build process. By default, git integration is globally disabled, simply change `enabled` to true in your `services.yaml` to enable it.
+## Git Integration ##
+Servicer provides a few handy Git integrations to optimize jobs. By default, git integration is disabled. To enable it, set `git: enabled` to `true`.
 
-```
-git:
-  enabled: false
-  config:
-    user.name: servicer
-  default-branch: master
-  ignore-unchanged: true
-  ignore-servicer-commits: false
-```
+### Change Detection ###
+Servicer can run `git diff` to determine which services in your project are different between the current commit and either the commit associated with its last successful job, or one passed to Servicer by an environment variable. This second commit is Servicer's _reference point_, and the list presented by `git diff` provides Servicer its change detection.
+
+By default, Servicer will maintain its own set of git tags that identify commits it should use as reference points. The format of these tags is `servicer-<BRANCH>-<BUILD_DATE>-<BUILD_NUMBER>`. Servicer relies upon the CI framework it is working on (like Jenkins) to provide the `BRANCH` and `BUILD_NUMBER` variables. Alternatively, the `GIT_DIFF_REF` environment variable can be set manually.
 
-#### Selective Builds ####
-For each service, you can optionally specify paths in your project that should trigger servicer to build, test, or deploy that service. Use `watch_paths` and `ignore_paths` to specify which files to match. Supported formats are simple text with `*` as a wildcard, or full regexes in the format `/.*/`. If no `watch_paths` or `ignore_paths` are provided, the service will be built every time.
+Servicer will use the first reference point it can find, and attempts to find it in this order:
+1. The `GIT_DIFF_REF` environment variable, if it’s set and is a valid commit.
+2. The latest servicer git tag matching the current branch, if `git: diff-tagging-enabled` is `true`.
+3. The latest servicer git tag matching any branch, if `git: diff-defaults-to-latest-tag` is `true`.
+
+### DRY Executions ###
+When using Git integration, Servicer will skip steps for the services that haven't changed. An example of how to employ this is below.
```

Comment:
To me `dry` means execute everything without any changes. I'd like to include a dry run feature in the future, but it would be independent of any git integration. I think it's sufficient to describe ignoring services as part of the complete Change Detection feature, and not use the `DRY Executions` here.